omod/src/main/webapp/resources/htmlforms/072-IntegratedMaternityRegister.xml

```diff
@@ -1,427 +1,668 @@
+                if ($viralLoadResults == VIRAL_LOAD_NOT_DETECTED_CONCEPT_ID) {
+                    $viralLoadField.val(1);
+                    $viralLoadField.attr('readonly', true).fadeTo(250, 0.65);
+                } else if($viralLoadResults == VIRAL_LOAD_DETECTED_CONCEPT_ID) {
+                    $viralLoadField.val(""); 
+                } else if ($viralLoadResults == VIRAL_LOAD_SAMPLE_REJECTED_CONCEPT_ID) {
+                    $viralLoadField.val("");
+                    $viralLoadField.attr('disabled', true);
+                } else {
+                    $viralLoadField.val("");
+                    $viralLoadField.attr('disabled', true);
+                }
+            };
+
+            var hivStatusChange = function () {
+                var $hivStatus = getValue('hiv-status.value');
+                var $hivInformationSection = jq(HIV_INFORMATION_SECTION);
+                var $arvsAdministeredToBabyField = jq(ARVS_ADMINISTERED_TO_BABY_FIELD).find('input');
+                if ($hivStatus == HIV_POSITIVE_CONCEPT_ID) {
+                    $hivInformationSection.find('input').removeAttr('disabled');
+                    $arvsAdministeredToBabyField.removeAttr('disabled');
+					$hivInformationSection.find('input,select').removeAttr('disabled');
+                } else {
+					$hivInformationSection.find('input,select').each(function() {										
+						if (jq(this).attr('id') != $hivStatusField.attr('id')) {
+							jq(this).attr('disabled', true);
+						} else {
+							jq(this).removeAttr('disabled');
+						}
+					});
+                    $arvsAdministeredToBabyField.attr('disabled', true);
+                }
+				viralLoadResultsChange();
+            };
+
+            var iycfChange = function() {
+                var $iycf =  getValue('iycf.value');
+                var $infantFeedingOptionField = getField('infant-feeding-option.value');
+                if ($iycf == YES_CONCEPT_ID) {
+                    $infantFeedingOptionField.removeAttr('disabled');
+                } else {
+                    $infantFeedingOptionField.attr('disabled', true);
+                }
+            };
+
+            var arvsToMotherChange =  function() {
+                var $arvsToMother =  getValue('arvs-to-mother.value');
+                var $preArtNumberField = jq(PRE_ART_NUMBER_FIELD).find('input:text');
+                if ($arvsToMother != '') {
+                    $preArtNumberField.removeAttr('disabled');
+                } else {
+                    $preArtNumberField.attr('disabled', true);
+                }
+            };
+
+            $viralLoadResultsField.change(viralLoadResultsChange);
+            $hivStatusField.change(hivStatusChange);
+            $iycfField.change(iycfChange);
+            $arvsToMotherField.change(arvsToMotherChange);
+			
+			viralLoadResultsChange();
+			hivStatusChange();
+			iycfChange();
+			arvsToMotherChange();
+
+        });
+
+
+        function enable_disable_inr(selected_option) {

```

Comment:
@moshonk Remove all old javascript which is probably broken. Even if the form exists we need to clean up this code.